include/lsst/afw/geom/SpherePoint.h

```diff
@@ -315,11 +315,6 @@ class SpherePoint final {
      *               circle defined by `bearing`
      * @returns a new point created by shifting this point
      *
-     * @throws pex::exceptions::DomainError
-     *         Thrown if `this.atPole()`.
-     * @throws pex::exceptions::InvalidParameterError
-     *         Thrown if `amount` is negative.
-     *
```

Comment:
If we're making all operations well-defined at the poles, do we still need `atPole`? IIRC it was originally created as a way to guard against these exceptions.

Reply:
Please update the description to give the behavior at the poles (depending on how the new code relates to `bearingTo`, this may be just a matter of removing the phrase "not at a coordinate pole").

Reply:
I believe `atPole` may still be useful, as some operations are not practical at the pole. One example: a user may query it to find out if longitude was set to 0 by the Point3D constructor; this could help avoid discontinuities in some situations.